Formula/cmocka.rb

```diff
@@ -15,12 +15,13 @@ class Cmocka < Formula
 
   def install
     args = std_cmake_args
+    args << "-DWITH_STATIC_LIB=ON" << "-DWITH_CMOCKERY_SUPPORT=ON" << "-DUNIT_TESTING=ON"
```

Comment:
I'm really not trying to be a problem and really appreciate all of the great work that homebrew does.

I think that `test do` catches homebrew installation issues. package unit tests, when they exist, should be run to confirm that the package is compiled correctly before a `bottle` is released.